src/components/AddToDoSection/AddToDoSection.tsx

```diff
@@ -11,15 +11,20 @@ const AddToDoSection = () => {
   const { addTaskToList } = useTaskStorage();
   const [taskTitle, setTaskTitle] = useState('');
 
-  // TODO: WHERE IS SAFE CHECK FOR EMPTY TASK TITLE? WHERE IS CLEARANCE OF INPUT AFTER ADDING TASK?
   const handleAddTaskButtonClick = () => {
-    addTaskToList({ title: taskTitle, id: Math.random().toString(), checked: false });
+    if (taskTitle !== '') {
```

Comment:
Technically, it is better to use action on check firstly and do not use else, as good practice - so it can be 

`
if(!taskTitle) { 
    window.alert('The task can not be empty')
    return;
    }
  ...rest of code that shoud happen when condition is met
`